src/node/element/mod.rs

```diff
@@ -229,19 +240,19 @@ implement! {
 
     #[doc = "A [`script`](https://www.w3.org/TR/SVG/script.html#ScriptElement) element."]
     struct Script [T: Into<String>] [inner, content: T] {
-        inner.append(::node::Text::new(content));
+        inner.append(crate::node::Text::new(content));
     }
 
     #[doc = "A [`style`](https://www.w3.org/TR/SVG/styling.html#StyleElement) element."]
     struct Style [T: Into<String>] [inner, content: T] {
-        inner.append(::node::Text::new(content));
+        inner.append(crate::node::Text::new(content));
     }
 }
 
 #[cfg(test)]
 mod tests {
-    use node::Node;
     use super::{Element, Style};
+    use crate::node::Node;
```

Comment:
Is it by rustfmt?

Reply:
No, It is by `cargo fix --edition` on commit d6159f4ef2c8fce81103cbc7ecabc4a0b01270b5